### ISSUE TITLE
A couple of issues with new version

### ISSUE THREAD

**Issue by Vane79** [Alias44/Nandonalts-Camping-Stuff#1]
Hello. So, I downloaded a version from here, made all the parts for a large tent, packed it, tried installing it - "install" creates a job, but no pawns would do it, and I can't prioritize it as well.

Also, I think I found a small incompatibility with either simple sidearms or inventory haul:
If a pawn picks up a tent pole but there is no place to store it, all the gizmos from pawns order bar disappear until the pawn drops the pole.

**Comment by Vane79** [comment]
Hello. So, I downloaded a version from here, made all the parts for a large tent, packed it, tried installing it - "install" creates a job, but no pawns would do it, and I can't prioritize it as well.

Also, I think I found a small incompatibility with either simple sidearms or inventory haul:
If a pawn picks up a tent pole but there is no place to store it, all the gizmos from pawns order bar disappear until the pawn drops the pole.
